README.md

```diff
@@ -34,6 +34,8 @@ Images and real-life data make text more engaging and powerful, but be sure what
 you use is appropriately licensed and available. Here again, even a rough idea
 for artwork can be polished by others.
 
+Pull requests are reviewed in markdown format. The 
+
```

Comment:
I think something is missing here :)

Reply:
whoops, I'll correct that in the next commit